### ISSUE TITLE
TightVNC viewer and this server

### ISSUE THREAD

**Issue by defuzer** [vadimpronin/guacamole-lite#29]
Hello should I be able to just start this repo then connect for example by TightVNC by typing ip_address:port ?

I opened this repo in Visual code then installed node modules and did nodemon ./index.js

**Comment by vadimpronin** [comment]
No, you can’t connect to Guacamole server with TightVNC. Guacamole itself is a VNC client meant to connect to remote VNC servers and you don’t need any clients on top of it except html5 browser

**Comment by defuzer** [comment]
Guacamole server (guacamole-lite) ipaddress is a individual depends of device? Or is always the same?

**Comment by vadimpronin** [comment]
This picture might help you to understand how it works
![arch](https://cloud.githubusercontent.com/assets/5534215/25705792/3140af24-30e7-11e7-99a0-0f77c5bf2e73.png)

Guacamole Server is a single server (with a single ip) that can connect to multiple computers via VNC or RDP.
And multiple web-based clients can connect to the Guacamole Server. 

Basically, the web-based client connects to Guacamole Server on IP x.x.x.x and tells it "connect me to a computer on IP y.y.y.y using VNC protocol and the following login and password".
Guacamole then goes on and connects to this computer on VNC protocol and acts as a proxy
